library/testing/TestSDKLoader.cxx

```diff
@@ -0,0 +1,114 @@
+#include <engine.h>
+#include <interactor.h>
+#include <loader.h>
+#include <window.h>
+
+#include <iostream>
+
+int TestSDKLoader(int argc, char* argv[])
+{
+  f3d::engine eng(f3d::engine::FLAGS_NONE);
+  f3d::loader& load = eng.getLoader();
+
+  std::string cowFilename = "cow.vtp";
```

Comment:
`constexpr std::string_view` is better when the string is immutable and known at build time.

Reply:
Indeed, I need to learn to use these.